mitosheet/src/mito/components/endo/ColumnHeaders.tsx

```diff
@@ -88,6 +89,11 @@ const ColumnHeaders = (props: {
                         e.preventDefault()
                         e.persist()
 
+                        // First we handle highlighting the border
```

Comment:
The `onDragOver` event handler is also triggered when we are resizing columns, so in order to only show the highlighted border when we're reordering columns, we need to case on the data transfer operation. You can see an example of this a few lines down in the `onDrop` function. 

https://github.com/mito-ds/monorepo/assets/18709905/731aa45d-85bd-4ec6-9647-5c68e8159d18

Reply:
Shoot - just realized (as is described in this comment:) https://github.com/mito-ds/monorepo/blob/f43b8fcab2c157325fc6c0b108679c9272e8039b/mitosheet/src/mito/components/endo/ColumnHeaders.tsx#L80) we won't be able to access the operation data from that function. Not sure what the best solution is for that one. Any ideas?

Reply:
At least since my recent change, the highlighted column won't stay there? Maybe I can check for operation in the border style function as a workaround? Or add the operation into the uistate object?

Reply:
Oh lol nevermind, there actually is a state object to track that in this file already. Just updated!